docs/machine-learning/resources/transforms.md

```diff
@@ -0,0 +1,131 @@
+| LpNormalizer | Normalize vectors (rows) individually by rescaling them to unit norm (L2, L1 or LInf). Performs the following operation on a vector X: Y = (X - M) / D, where M is mean and D is either L2 norm, L1 norm or LInf norm. |
+| MeanVarianceNormalizer | Normalizes the data based on the computed mean and variance of the data. |
+| MinMaxNormalizer | Normalizes the data based on the observed minimum and maximum values of the data. | 
+| SupervisedBinNormalizer | Similar to BinNormalizer, but calculates bins based on correlation with the label column, not equi-density. The new value is bin_number / number_of_bins. |
+
+## Row Filters
+
+| Transform | Definition |
+| --- | --- |
+| RowRangeFilter | Filters a dataview on a column of type Single, Double or Key (contiguous). Keeps the values that are in the specified min/max range. NaNs are always filtered out. If the input is a Key type, the min/max are considered percentages of the number of values. |
+| RowSkipAndTakeFilter | Allows limiting input to a subset of rows at an optional offset. Can be used to implement data paging. |
+| RowSkipFilter	| Allows limiting input to a subset of rows by skipping a number of rows. |
+| RowTakeFilter	| Allows limiting input to a subset of rows by taking N first rows. |
+
+## Schema
+
+| Transform | Definition |
+| --- | --- |
+| ColumnConcatenator | Concatenates two columns of the same item type. |
+| ColumnCopier | Duplicates columns from the dataset.|
+| ColumnDropper	| Drops columns from the dataset. |
+| ColumnSelector | Selects a set of columns, dropping all others. |
+| ColumnTypeConverter | Converts a column to a different type, using standard conversions. |
+| KeyToTextConverter | KeyToValueTransform utilizes KeyValues metadata to map key indices to the corresponding values in the KeyValues metadata. |
+| NGramTranslator | Produces a bag of counts of ngrams (sequences of consecutive values of length 1-n) in a given vector of keys. It does so by building a dictionary of ngrams and using the id in the dictionary as the index in the bag. | 
+| OptionalColumnCreator	| If the source column does not exist after deserialization, create a column with the right type and default values. | 
+
+## Text processing and featurization
+
+| Transform | Definition |
+| --- | --- |
+| CharacterTokenizer | Character-oriented tokenizer where text is considered a sequence of characters. |
+| TextFeaturizer | A transform that turns a collection of text documents into numerical feature vectors. The feature vectors are normalized counts of (word and/or character) ngrams in a given tokenized text. |
+| TextToKeyConverter | Converts input values (words, numbers, etc.) to index in a dictionary. |
+| WordTokenizer	| The input to this transform is text, and the output is a vector of text containing the words (tokens) in the original text. The separator is space, but can be specified as any other character (or multiple characters) if needed. |
```

Comment:
is space --> is a space, 
can be specified as any other character (or multiple characters) if needed. --> any other character (or multiple characters) can be specified.